tests/storage/nfs/test_nfs_sr.py

```diff
@@ -29,6 +36,40 @@ def test_vdi_is_not_open(self, dispatch_nfs):
         vdi = dispatch_nfs
         assert not vdi_is_open(vdi)
 
+    @pytest.mark.small_vm
+    @pytest.mark.usefixtures('hostA2')
+    # Make sure this fixture is called before the parametrized one
+    @pytest.mark.usefixtures('vm_ref')
+    @pytest.mark.parametrize('dispatch_nfs', ['vm_on_nfs_sr'], indirect=True)
+    def test_plugin_nfs_on_on_slave(self, dispatch_nfs: 'VM'):
+        vm = dispatch_nfs
+        vm.start()
+        vm.wait_for_os_booted()
+        host = vm.get_residence_host()
+
+        vdi = VDI(vm.vdi_uuids()[0], host=host)
```

Comment:
Usually,  we use the pool master for the VDI `host` parameter.
But here I don't thing it's a problem.